src/js/timeline/Timeline.js

```diff
@@ -527,8 +528,13 @@ class Timeline {
         this._menubar.changeVisibleTicks(e.visible_ticks);
     }
 
+    _onForwardToEnd(e) {
+        this.goToEnd();
+        this.fire("forward_to_end", { unique_id: this.current_id }, this);
+    }
+
     _onBackToStart(e) {
-        this._storyslider.goTo(0);
+        this.goToStart();
```

Comment:
We already had the `goToStart` method but just didn't use it :slightly_smiling_face: